website/content/en/docs/contribution-guidelines/release.md

```diff
@@ -298,36 +294,36 @@ To verify binaries and tags, see the [verification section](#verifying-a-release
 `ansible-operator` and `helm-operator` release binaries and signatures are similarly built for upload so `make run`
 can download them in their respective operator type projects. See [#3327](https://github.com/operator-framework/operator-sdk/issues/3327) for details.
 
-Push tag `v1.3.0` upstream, assuming `origin` is the name of the upstream remote:
+Push tag `v1.3.0` upstream, assuming `upstream` is the name of the upstream remote:
 
 ```sh
-$ git push origin v1.3.0
+$ git push upstream v1.3.0
```

Comment:
```suggestion
git push upstream v1.3.0
```

Reply:
could we do the same across the doc?

Reply:
I can, this was the way they were originally, I just moved them.